babel.config.js

```diff
@@ -1,11 +1,12 @@
 module.exports = {
-    presets: [['zillow', { modules: false }]],
-    env: {
-        cjs: {
-            presets: ['zillow'],
-        },
-        test: {
-            presets: ['zillow'],
-        },
-    },
+    presets: [
+        [
+            'babel-preset-zillow',
+            {
+                targets: {
+                    node: 'current',
```

Comment:
As a CLI-only program, there's no reason to transpile for the browser.

Reply:
I think I had it as this to mirror the configuration of a CRS project -- to "dogfood" our own tooling.